[2025-02-18T04:32:20.467+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-18T04:32:20.489+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-11-01T00:00:00+00:00 [queued]>
[2025-02-18T04:32:20.500+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_first_dag.hello_task scheduled__2024-11-01T00:00:00+00:00 [queued]>
[2025-02-18T04:32:20.506+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-18T04:32:20.542+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): hello_task> on 2024-11-01 00:00:00+00:00
[2025-02-18T04:32:20.557+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=1024) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-18T04:32:20.560+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'my_first_dag', 'hello_task', 'scheduled__2024-11-01T00:00:00+00:00', '--job-id', '260', '--raw', '--subdir', 'DAGS_FOLDER/my-first-dag.py', '--cfg-path', '/tmp/tmpseaj0d_r']
[2025-02-18T04:32:20.566+0000] {standard_task_runner.py:105} INFO - Job 260: Subtask hello_task
[2025-02-18T04:32:20.562+0000] {standard_task_runner.py:72} INFO - Started process 1036 to run task
[2025-02-18T04:32:20.595+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:209 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:32:20.634+0000] {task_command.py:467} INFO - Running <TaskInstance: my_first_dag.hello_task scheduled__2024-11-01T00:00:00+00:00 [running]> on host 3420485c558b
[2025-02-18T04:32:20.636+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/settings.py:470 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-02-18T04:32:20.713+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='my_first_dag' AIRFLOW_CTX_TASK_ID='hello_task' AIRFLOW_CTX_EXECUTION_DATE='2024-11-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-01T00:00:00+00:00'
[2025-02-18T04:32:20.715+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-18T04:32:20.725+0000] {logging_mixin.py:190} INFO - Hello, Airflow!
[2025-02-18T04:32:20.725+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-18T04:32:20.735+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-18T04:32:20.737+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=my_first_dag, task_id=hello_task, run_id=scheduled__2024-11-01T00:00:00+00:00, execution_date=20241101T000000, start_date=20250218T043220, end_date=20250218T043220
[2025-02-18T04:32:20.807+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-18T04:32:20.827+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-18T04:32:20.830+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
